app/classifier/tasks/survey/index.spec.js

```diff
@@ -5,68 +5,68 @@ import SurveyTask from './';
 import Choice from './choice';
 import { workflow } from '../../../pages/dev-classifier/mock-data';
 
-describe('Survey Task', function(){
+describe('Survey Task', () => {
```

Comment:
Arrow functions are actually discouraged in mocha: https://mochajs.org/#arrow-functions

But with these tests, I don't think it matters. Just an FYI.

Reply:
TIL, thanks @srallen

Reply:
Ah okay, interesting! I'll have to hack my linter for personal projects now...